app/mailers/notification_mailers/base.rb

```diff
@@ -29,11 +29,12 @@ def name_and_address(name, email)
     end
 
     private
+
     def default_headers
       headers = {
-        :from => AppConfig.mail.sender_address.get,
-        :host => "#{AppConfig.pod_uri.host}",
-        :to => name_and_address(@recipient.name, @recipient.email)
+        from: AppConfig.mail.sender_address.get,
+        host: "#{AppConfig.pod_uri.host}",
+        to:   name_and_address(@recipient.name, @recipient.email)
```

Comment:
@salbertson Hound seems to ignore our [rubocop.yml](https://github.com/diaspora/diaspora/blob/develop/.rubocop.yml#L34) here.